chainer/links/theano/theano_function.py

```diff
@@ -35,7 +35,13 @@ class TheanoFunction(link.Link):
     and backward calculation from inputs and ouptuts. And then, it sends data
     in :class:`chainer.Variable` to the function and gets results from Theano.
 
-    .. admonition:: Example
+    .. rubric:: Example
+
+    .. testcode::
+       # See chainer/chainer#5997
+       :skipif: os.environ.get('READTHEDOCS') != 'True' \
```

Comment:
In the readthedocs build, this `skipif` should never be effective, because that would cause the code to disapper from the documentation.